frontend/src/theme/primitives/spacing.ts

```diff
@@ -31,4 +31,15 @@ export const spacing = {
   twoXL: "1.5rem",
   threeXL: "2rem",
   fourXL: "4rem",
+  nonePx: 0,
```

Comment:
there is already `none` above which also is just 0

Reply:
That is true but it's a string. I guess we could technically just cast none as a number but I thought this would be better?